docs/csharp/language-reference/keywords/static.md

```diff
@@ -10,9 +10,11 @@ ms.assetid: 5509e215-2183-4da3-bab4-6b7e607a4fdf
 ---
 # static (C# Reference)
 
+This page covers the `static` modifier keyword. The `static` keyword is also part of the [using static](using-static.md) directive.
```

Comment:
Should there be a similar note in the *using.md* file?

Reply:
This is already a note in the *using.md* that links to *using-static.md*.